rts/Rml/Backends/RmlUi_Backend.cpp

```diff
@@ -413,40 +435,41 @@ bool RmlGui::ProcessEvent(const SDL_Event &event)
 	{
 		return false;
 	}
-	SDL_Event ev(event);
-	switch (ev.type)
-	{
-	case SDL_WINDOWEVENT:
+	for (auto &context : data->contexts)
 	{
```

Comment:
consider keeping the way the function looked previously as `ProcessOneContext` or such, and then here you'd just do `for (auto &context : contexts) ProcessOneContext(context);`

Reply:
:+1: